index.bs

```diff
@@ -6826,8 +6824,8 @@ This [=client extension|client=] [=registration extension=] facilitates reportin
         ::  This OPTIONAL property is a [=human palatability|human-palatable=] description of the credential's [=managing authenticator=],
             chosen by the user.
 
-            The [=client=] MUST allow the user to choose this value,
-            MAY or MAY not present that choice during [=registration ceremonies=],
+            During [=registration ceremonies=] the [=client=] MUST allow the user to choose this value,
+            MAY or MAY not present that choice,
             and MAY reuse the same value for multiple credentials with the same [=managing authenticator=] across multiple [=[RPS]=].
```

Comment:
"MUST allow the user to choose this value, MAY or MAY not present that choice [at all]" sounds internally inconsistent to me - how could the user choose if never presented with a choice? Perhaps we should rephrase this a bit to make the intended meaning a little clearer, how about this?

```suggestion
            The [=client=] MUST allow the user to choose this value.
            That choice MAY be presented during the [=registration ceremony|registration=] or [=authentication ceremony=]
            or MAY be made available outside the ceremony, for example in client settings.
            The [=client=] MAY reuse the same value for multiple credentials with the same [=managing authenticator=] across multiple [=[RPS]=].
```

Reply:
I like that wording a lot better, I've updated this part.